app/models/nonbillable_account.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class NonbillableAccount < Account
+  before_validation :set_owner, :set_description, :set_created_by, :set_account_number, :set_expries_at
+
+  # Since this account can be used by anyone, we only need one in the system
+  # so this class method should be used to access it.
+  def self.account
+    @account ||= first || create
+  end
+
+  def account_open?(_)
```

Comment:
Can you use a more descriptive unused variable name here?  What's getting passed in?